File: Lista/Lista Estática Circular/Lista Estática Java/TesteLista.java
public class TesteLista {

    public static void main(String[] args) {
        Listavel lista = new Listando();

        lista.anexar("Caio");
        lista.anexar("Cotrim");
        lista.inserir(1, "Pereira");
        System.out.println(lista.imprimir());

        lista.anexar("Pereira");
        lista.apagar(0);
        lista.inserir(0, "Cotrim");
        lista.inserir(2, "Caio");
        System.out.println(lista.imprimir());

        System.out.println(lista.selecionar(1));

        lista.atualizar(1, "G");
        System.out.println(lista.imprimir());
    }
}

File: Unidade I/Lista/Lista Estática Circular/Lista Estática Kotlin/TesteLista.kt
class TesteLista {
    fun main() {
        var lista: Listavel = ListaEstaticaCircular()
        lista.anexar("Caio")  
        lista.anexar("Cotrim")   
        lista.inserir(1,"Pereira")
        println("${lista.imprimir()}")
        lista.anexar("Pereira")   
        lista.apagar(0) 
        lista.inserir(0,"Cotrim")   
        lista.inserir(2,"Caio");	
        println("${lista.imprimir()}") 
        println("${lista.selecionar(1)}")
        lista.atualizar(1,"G")
        println("${lista.imprimir()}")  
    }
}    